src/python/bot/fuzzers/options.py

```diff
@@ -160,19 +160,22 @@ def get_ubsan_options(self):
 def get_fuzz_target_options(fuzz_target_path):
   """Return a FuzzerOptions for the given target, or None if it does not
   exist."""
-  options_file_path = fuzzer_utils.get_supporting_file(fuzz_target_path,
-                                                       OPTIONS_FILE_EXTENSION)
-  options_cwd = os.path.dirname(options_file_path)
+  file_path = fuzzer_utils.get_supporting_file(fuzz_target_path,
```

Comment:
What's the reason for the changes here? It seems like we're just effectively renaming a variable? I prefer the old one tbh.